src/gtc_unstructured/cpp_util/unstructured/include/gridtools/usid/atlas.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <atlas/mesh/Connectivity.h>
+
+#include <gridtools/storage/builder.hpp>
+
+namespace atlas::mesh {
```

Comment:
```suggestion
namespace gridtools::atlas {
```
?

Reply:
`make_storage_producer` should be in `atlas::mesh` together with atlas::mesh::Connectivity to ensure ADL.